diff --git a/cupidoBackendInterfaces/src/unibo/as/cupido/backendInterfaces/common/ObservedGameStatus.java b/cupidoBackendInterfaces/src/unibo/as/cupido/backendInterfaces/common/ObservedGameStatus.java
index 192d184..027e18a 100644
--- a/cupidoBackendInterfaces/src/unibo/as/cupido/backendInterfaces/common/ObservedGameStatus.java
+++ b/cupidoBackendInterfaces/src/unibo/as/cupido/backendInterfaces/common/ObservedGameStatus.java
@@ -14,12 +14,25 @@ public class ObservedGameStatus implements Serializable {
 	private static final long serialVersionUID = 1L;
 
 	public PlayerStatus[] ogs;
+	
+	/**
+	 * The index of the first player that dealt a card in the current trick.
+	 * When this is 0, the bottom player led the trick, when it's 1 the trick was
+	 * led by the left player, and so on for other players, in clockwise order.
+	 * 
+	 * If there is currently no trick, this is -1.
+	 * This can happen in four cases: when some players are still missing in the table,
+	 * when players are passing cards, when the players have passed cards but no-one
+	 * has dealt the two of clubs yet, and when the game has ended.
+	 */
+	public int firstDealerInTrick;
 
 	public ObservedGameStatus() {
 
 	}
 
-	public ObservedGameStatus(PlayerStatus[] players) {
+	public ObservedGameStatus(PlayerStatus[] players, int firstDealerInTrick) {
 		ogs = players;
+		this.firstDealerInTrick = firstDealerInTrick;
 	}
 }
